cabal.project

```diff
@@ -14,6 +14,13 @@ source-repository-package
     tag: 2547ad1e80aeabca2899951601079408becbc92c
     --sha256: 1p2kg2w02q5w1cvqzhfhqmxviy4xrzada3mmb096j2n6hfr20kri
 
+source-repository-package
+    type: git
+    location: https://github.com/hamishmack/foundation
```

Comment:
Because this build fix is so small, could we do it as a patch instead of forking on github?

Reply:
I looked into this and it won't work as a patch.  We need `cabal configure` to see the patched version and haskell.nix can only patch the sources selected after `cabal configure` runs.

Reply:
OK - fair enough!